app/src/main/java/com/keylesspalace/tusky/components/timeline/viewmodel/NetworkTimelineViewModel.kt

```diff
@@ -232,7 +232,7 @@ class NetworkTimelineViewModel @Inject constructor(
     }
 
     override fun clearWarning(status: StatusViewData.Concrete) {
-        updateActionableStatusById(status.actionableId) {
```

Comment:
The actionableId was never right here?

Reply:
It's because `updateActionableStatusById` looks up the status by the *outer* id, then updates the *actionable* status of that status